--- sqlite-src-3420000.orig/test/tester.tcl	2023-05-16 15:45:19.000000000 +0200
+++ sqlite-src-3420000/test/tester.tcl	2023-07-28 16:14:56.595677285 +0200
@@ -213,6 +213,8 @@ proc do_copy_file {force from to} {
   #
   # The solution is to wait a short amount of time before retrying the copy.
   #
+  set from "/tmp/$from"
+  set to "/tmp/$to"
   if {$nRetry > 0} {
     for {set i 0} {$i<$nRetry} {incr i} {
       set rc [catch {
@@ -287,6 +289,7 @@ proc do_delete_file {force args} {
     # The solution is to wait a short amount of time before retrying the
     # delete.
     #
+    set filename "/tmp/${filename}"
     if {$nRetry > 0} {
       for {set i 0} {$i<$nRetry} {incr i} {
         set rc [catch {
@@ -613,7 +616,7 @@ proc reset_db {} {
   forcedelete test.db
   forcedelete test.db-journal
   forcedelete test.db-wal
-  sqlite3 db ./test.db
+  sqlite3 db test.db
   set ::DB [sqlite3_connection_pointer db]
   if {[info exists ::SETUP_SQL]} {
     db eval $::SETUP_SQL
